### DISCUSSION TITLE
Minified distribution link for module build?

### DISCUSSION THREAD

**Discussion by simonferndriger** [alpinejs/alpine#3533] [4. General]
Is there perhaps a minified distribution link for the module buil available, e.g. for `https://cdn.jsdelivr.net/npm/alpinejs@3.12.0/dist/module.esm.js`?

**Comment by SimoTod** [comment]
https://cdn.jsdelivr.net/npm/alpinejs@3.12.0/dist/module.esm.min.js

**Reply by SimoTod** [reply]
jsdelivr minifies any js file if you just use .min.js at the end instead of .js

**Reply by ekwoka** [reply]
Additionally, for module use, `https://esm.sh/alpinejs`

esm.sh can dynamically bundle modules from anything, and you can tell it if you wan some to be external (imported as separate modules, with you even controlling the versions used) or bundled in.

It's the most proper full feature module CDN.

**Reply by simonferndriger** [reply]
Thanks, how does this work exactly? Is there a documentation on this? Sorry, I'm new to this...

**Reply by ekwoka** [reply]
Esm.sh?

You can just go to https://esm.sh to see

**Comment by simonferndriger** [comment]
Awesome, thank you!
